main.go

```diff
@@ -55,6 +55,8 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
+	var crunchybridgeAPIURL string
+	flag.StringVar(&crunchybridgeAPIURL, "crunchybridge-crunchybridgeapi-url", "https://api.crunchybridge.com", "the Crunchy bridge API URL (no slash in the end).")
```

Comment:
How about `crunchybridgeapi-url` to remove the stutter and shorten the flag?

Reply:
make sense, thanks and changed